LICENSE.txt

```diff
@@ -200,3 +200,824 @@
    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
    See the License for the specific language governing permissions and
    limitations under the License.
+
+   This distribution has a binary dependency on jibber_jabber (https://github.com/cloudfoundry-attic/jibber_jabber),
+   which is available under Apache License, Version 2. (as above)
+
+   This distribution has a binary dependency on afero (https://github.com/spf13/afero),
+   which is available under Apache License, Version 2. (as above)
+
+   This distribution has a binary dependency on cobra (https://github.com/spf13/cobra),
+   which is available under Apache License, Version 2. (as above)
+
+   This distribution has a binary dependency on mousetrap (https://github.com/inconshreveable/mousetrap),
+   which is available under Apache License, Version 2. (as above)
+
+   This distribution has a binary dependency on stacktrace (https://github.com/palantir/stacktrace),
+   which is available under Apache License, Version 2. (as above)
+
+   This distribution has a binary dependency on yaml (https://github.com/go-yaml/yaml),
+   which is available under Apache License, Version 2. (as above)
+
+   This distribution has a binary dependency on properties (https://github.com/magiconair/properties),
+   which is available under a "2-clause BSD" license. (as below)
+
+   goproperties - properties file decoder for Go
```

Comment:
how about we add a link to this license file here https://github.com/magiconair/properties/blob/master/LICENSE